src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo_Metadata.cs

```diff
@@ -407,10 +422,18 @@ private void GenerateMetadataNodes()
                         // we just pull in methods that have attributes on them.
```

Comment:
i wonder if we can do better.  at least see if hte attribute is called "Extension".

Reply:
Sure can do, it's just unclear how much benefit it would provide

Reply:
it would probably greatly decrease the number of methods we pull in.

Reply:
this still seems worth looking into IMO.

Reply:
this doesn't seem to cause any issues during my test in roslyn.sln. Will file a bug and consider this in a follow-up PR